exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,18 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function upCase(name) {
+    return name.toUpperCase();
+}
+function greeting(name) {
+    let shouty = upCase(name);
+    let message = "GOOD DAY " + shouty;
+    return message;
+}
+console.log(greeting(mentor1));
+console.log(greeting(mentor2));
+console.log(greeting(mentor3));
+console.log(greeting(mentor4));
+console.log(greeting(mentor5));
+
```

Comment:
Great simple way of doing this project.

Just suggesting alternatives, all the console.logs could have been done in the greeting as well but well done.